example/unicorn/templates/unicorn/crispy.html

```diff
@@ -0,0 +1,14 @@
+{% extends "www/base.html" %}
+{% load crispy_forms_tags unicorn %}
+
+{% load crispy_forms_tags %}
+
+{% block content %}
+
+<div unicorn:view>
+  <h2>Crispy</h2>
+
+  {% crispy view.form_class view.form_class.helper %}
```

Comment:
the `view.form_class.helper` can be omitted (if FormHelper attr is called `helper`, as you did)